lib/rpmsg/rpmsg.c

```diff
@@ -257,7 +257,7 @@ void rpmsg_destroy_ept(struct rpmsg_endpoint *ept)
 		return;
 
 	rdev = ept->rdev;
-	if (rdev->support_ns && ept->addr != RPMSG_NS_EPT_ADDR)
+	if (ept->name[0] && rdev->support_ns && ept->addr != RPMSG_NS_EPT_ADDR)
```

Comment:
The same comment as before.

Reply:
The check is required since the nameless endpoint have the valid local address but RPMSG_NS_DESTROY shouldn't send out.